legion/legion/edi/server.py

```diff
@@ -328,6 +348,7 @@ def load_cluster_config(application):
                                                            application.config['CLUSTER_SECRETS']['grafana.user'],
                                                            application.config['CLUSTER_SECRETS']['grafana.password'])
     application.config['GRAFANA_CLIENT'] = grafana_client
+    application.config['JWT_CONFIG'] = legion.k8s.load_secrets(application.config['JWT_CONFIG_PATH'])
```

Comment:
Can we use secret mounting here?

Reply:
Secret is mounted and path to a mount is passed via Environment variable